tools/state-copy/copy.mjs

```diff
@@ -0,0 +1,73 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
```

Comment:
The state copy tool isn't used, so I am happy to remove, but it works and feels useful enough to leave it.

Reply:
I Agree With This Comment